src/stepper/Stepper.tsx

```diff
@@ -203,60 +185,39 @@ export default class Stepper extends PureComponent<StepperProps, StepperState> {
     const buttonSize = StepperButtonSize[display]
 
     return (
-      <div className={cc(['kirk-stepper', `kirk-stepper-${display}`, className])}>
-        <input
-          tabIndex={0}
-          className="kirk-stepper-range"
-          type="range"
-          min={this.props.min}
-          max={this.props.max}
-          defaultValue={`${this.state.value}`}
-          step={this.props.step}
-          aria-valuemin={this.props.min}
-          aria-valuemax={this.props.max}
-          aria-valuenow={this.state.value}
-          onKeyDown={this.handleKeyDown}
-        />
+      <div
+        className={cc(['kirk-stepper', `kirk-stepper-${display}`, className])}
+        aria-label={children}
```

Comment:
Does this aria label on this div ever get announced ?

Reply:
Yes in browsing mode, it focuses the div and read the label.